hotsos/core/plugins/openstack/nova.py

```diff
@@ -1,6 +1,7 @@
 import os
 import re
 
+from hotsos.core.plugins.kernel.sysfs import CPU
```

Comment:
I guess we don't enforce alphabetical ordering, or some other things, with pep8, it's tree-wide so not just here. I had been looking at some pep8/pylint cleanups so can add some of this there.

Reply:
we don't but i'm all for it though so welcome pylint tweaks to enforce it